langlib/lang.stream/src/main/ballerina/src/lang.stream/stream.bal

```diff
@@ -86,7 +86,7 @@ public isolated function 'map(stream<Type,ErrorType> stm, @isolatedParam functio
 # + func - combining function
 # + initial - initial value for the first argument of combining function
 # + return - result of combining the members of `stm` using the combining function
-public isolated function reduce(stream<Type,ErrorType> stm, @isolatedParam function(Type1 accum, Type val) returns Type1 func, Type1 initial)
+public function reduce(stream<Type,ErrorType> stm, function(Type1 accum, Type val) returns Type1 func, Type1 initial)
```

Comment:
Created https://github.com/ballerina-platform/ballerina-lang/issues/26575 to track fixing these.